addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4764,6 +4764,31 @@ registry.DynamicSvg = SnippetOptionWidget.extend({
     _computeVisibility(methodName, params) {
         return this.$target.is("img[src^='/web_editor/shape/']");
     },
+    /**
+     * Returns the default colors for the currently selected SVG.
+     *
+     * @private
+     */
+    _getDefaultColors() {
+        const svgSrc = this.$target.attr("src");
+        if (!svgSrc) {
+            return {};
+        }
+        const url = new URL(svgSrc, window.location.origin);
+        return Object.fromEntries(url.searchParams.entries());
+    },
+    /**
+     * @override
+     */
+    _renderCustomXML(uiFragment) {
+        Object.keys(this._getDefaultColors()).map(colorName => {
+            if (colorName.charAt(0) == 'c') {
```

Comment:
see previous comment, shoudl already have only color here

Reply:
The DynamicSvg widget seems to be related to all its colors, therefore this rendering patches the DOM to include the editor for each of them